pymc/distributions/distribution.py

```diff
@@ -83,6 +85,21 @@
 PLATFORM = sys.platform
 
 
+class MomentRewrite(GraphRewriter):
+    def add_requirements(self, fgraph):
+        fgraph.attach_feature(ReplaceValidate())
+
+    def apply(self, fgraph):
+        for node in fgraph.toposort():
+            for i, inp in enumerate(node.inputs):
+                if (
+                    hasattr(inp, "owner")
+                    and hasattr(inp.owner, "op")
+                    and isinstance(inp.owner.op, RandomVariable)
```

Comment:
Ah could also be a symbolic one!

```suggestion
                    and isinstance(inp.owner.op, (RandomVariable, SymbolicRandomVariable))
```

Reply:
Thank you!